packages/babel-compiler/package.js

```diff
@@ -0,0 +1,17 @@
+Package.describe({
+  name: "babel-compiler",
+  summary: "Parser/transpiler for ECMAScript 6+ syntax",
+  // Tracks the npm version below.  Use wrap numbers to increment
+  // without incrementing the npm version.
+  version: '5.6.17'
```

Comment:
I think we should version this package in step with `meteor-babel`, instead of `babel-core`, since a particular version of `meteor-babel` corresponds to a pegged version of `babel-core` anyway.